src/components/ClientInfo.js

```diff
@@ -17,7 +17,9 @@ import LoadingProgress from './LoadingProgress'
 
 import { selectCurrencySymbol } from '../scripts/selectors'
 import { lightGrey } from '../styles/colors.scss'
+import {ClientInfoName} from '../styles/ClientInfoName.scss'
```

Comment:
Fix typo  `ClientInfoName.scss` to `ClientInfo.scss`

Reply:
done already